djangoproject/templates/accessibility/index.html

```diff
@@ -0,0 +1,150 @@
+
+  <h2>{% translate "Our commitment" %}</h2>
+
+  <p>
+    {% blocktranslate trimmed %}
+      We strive to make Django itself and its related online and in-person spaces
+      accessible for all. This includes:
+    {% endblocktranslate %}
+  </p>
+
+  <ul class="simple">
+    <li>{% translate "Supporting the creation of accessible web applications with Django." %}</li>
+    <li>
+      {% blocktranslate trimmed %}
+        Ensuring that our documentation, community tools, and websites are navigable and usable for
+        people with disabilities.
+      {% endblocktranslate %}</li>
+    <li>
+      {% blocktranslate trimmed %}
+        Encouraging accessibility awareness across all aspects of the project, from events and
+        communication to software design and development.
+      {% endblocktranslate %}</li>
+  </ul>
+
+  <h3>{% translate "Accessibility in the Django web framework" %}</h3>
+
+  <p>
+    {% blocktranslate trimmed %}
+      We aim for conformance with recognized standards such as the
+      <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">
+        Web Content Accessibility Guidelines</a>
+      (WCAG) and the
+      <a href="https://www.w3.org/WAI/standards-guidelines/atag/">
+        Authoring Tool Accessibility Guidelines</a>
+      (ATAG). Where feasible, we strive to exceed these standards. Through documentation, tutorials,
+      and exemplar implementation, we highlight best practices for building accessible applications.
+    {% endblocktranslate %}
+  </p>
+
+  <h3>{% translate "Community and events" %}</h3>
+
+  <p>
+    {% blocktranslate trimmed %}
+      The Django Software Foundation is committed to ensuring that events receiving our grant funding
+      are accessible to all. We encourage organizers to choose venues and platforms that meet physical
+      accessibility needs, that provide resources such as captioning and sign language interpretation
+      where possible. We welcome feedback to make improvements on any and all aspects of physical
+      accessibility.
+    {% endblocktranslate %}
+  </p>
+
+  <h2>{% translate "What we do" %}</h2>
+
+  <h3>{% translate "Known issues and continuous improvement" %}</h3>
+
+  <p>
+    {% blocktranslate trimmed %}
+      We acknowledge that there is always room to improve. Accessibility is an ongoing process, and
+      we aim to address issues as they arise. Known accessibility issues with the Django framework,
+      admin interface, or community platforms are tracked publicly, and we welcome collaboration to
+      resolve them.
+    {% endblocktranslate %}
+  </p>
+
+  <p>{% translate "For further details, see:" %}</p>
+
+  <ul class="simple">
+    <li>
+      <a href="https://code.djangoproject.com/query?description=~accessibility\&status=assigned\
+               &status=closed\&status=new\&or\&keywords=~accessibility\&or\&summary=~accessibility\&order=id\
+               &desc=1\&col=id\&col=summary\&col=owner\&col=type\&col=component">
```

Comment:
```suggestion
      <a href="https://code.djangoproject.com/query?description=~accessibility&status=assigned&status=closed&status=new&or&keywords=~accessibility&or&summary=~accessibility&order=id&desc=1&col=id&col=summary&col=owner&col=type&col=component">
```

URL is off (was off in the live version too!), and I don’t think we’re really gaining anything by breaking this into multiple lines, just makes it harder to update the content.